Doc/library/math.rst

```diff
@@ -447,20 +460,14 @@ Number-theoretic and representation functions
    .. versionadded:: 3.9
 
 
-Note that :func:`frexp` and :func:`modf` have a different call/return pattern
-than their C equivalents: they take a single argument and return a pair of
-values, rather than returning their second return value through an 'output
+Note that :func:`frexp` has a different call/return pattern
+than its C equivalents: it takes a single argument and return a pair of
+values, rather than returning its second return value through an 'output
```

Comment:
Same as for note about modf.

Reply:
I moved the notes nor now. See https://github.com/python/cpython/issues/126722 for later.